app/utils/touch.js

```diff
@@ -15,29 +15,32 @@ class Touch extends React.Component {
 
 	render() {
 		const {
-			children, onPress, theme, underlayColor, ...props
+			children, onPress, onLongPress, theme, style, ...props
 		} = this.props;
 
 		return (
-			<RectButton
+			<Pressable
```

Comment:
Why is this necessary?
That's a serious refactor and I'm not sure Pressable is that stable already.